docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -1,78 +1,72 @@
 ---
 title: Database Access GUI Clients
-description: How to configure graphical database clients
+description: How to configure graphical database clients for Teleport Database Access.
 ---
 
 # Graphical Database Clients
 
-This page provides instructions on how to configure various graphical database
-clients to work with Teleport Database Access.
+This article describes how to configure popular graphical database clients to work with Teleport Database Access.
 
-Pick the client you're interested in from the table of contents on the right.
+## Obtain local certificate/key-pair
 
-## pgAdmin 4
+Issue the following command after you login through `tsh`:
```

Comment:
I think this is more helpful to have this above since both clients will require this step to get the cert files.